tests/resource_test.js

```diff
@@ -0,0 +1,65 @@
+/// <reference path="./steps.d.ts" />
+Feature('Resource test');
+
+var FirstLogin = false;
```

Comment:
I think these are unneeded variables copied from my test - you should try your test without them. I should go back and see if they are used anywhere in mine as well.